java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -168,6 +168,7 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
                         "Catania",
                         new GeospatialData(15.087269, 37.502669)));
         baseTransaction.geopos(geoKey1, new String[] {"Palermo", "Catania"});
+        baseTransaction.geodist(geoKey1, "Palermo", "Catania");
```

Comment:
test `geodist` with unit

Reply:
addressed in the latest commit with other comments, thanks!